tests/load-test.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# Test if the Pulsar.node built from ../pkg/linux/build-napi-inside-docker.sh
+# can be loaded in other Linux distributions.
+
+set -e
+
+if [[ $# -lt 2 ]]; then
+    echo "Usage $0 <node-image-name> <platform>
+
+See https://hub.docker.com/_/node for valid images"
+    exit 1
+fi
+IMAGE=$1
+PLATFORM=$2
+
+ROOT_DIR=${ROOT_DIR:-$(git rev-parse --show-toplevel)}
+cd $ROOT_DIR
+
+if [[ ! -f $ROOT_DIR/pulsar-client-node.tar.gz ]]; then
+    git archive -o pulsar-client-node.tar.gz HEAD
+fi
```

Comment:
If this script is not only used in the CI, we need to remove `pulsar-client-node.tar.gz` after executing. If this script is only for CI. I am +1 for moving it  to `pkg` folder.

Reply:
We can run the script locally. For example, assuming there is a new Node.js image (e.g. `node-20-xxx`), you can run `./tests/load-test.sh node-20-xxx x86_64` to verify the `Pulsar.node` built by `pkg/linux/build-napi-inside-docker.sh` can work for that.

Here I don't remove `pulsar-client-node.tar.gz` because it can be reused for later tests.

Reply:
Oh I realized the problem. We should not reuse the `pulsar-client-node.tar.gz` because it might be built with an earlier version. I will modify it.